src/main/scala/firrtl/annotations/AnnotationUtils.scala

```diff
@@ -3,9 +3,15 @@
 package firrtl
 package annotations
 
+import net.jcazevedo.moultingyaml._
+import firrtl.annotations.AnnotationYamlProtocol._
+
 import firrtl.ir._
 
 object AnnotationUtils {
+  def toYaml(a: Annotation): String = a.toYaml.prettyPrint
+  def fromYaml(s: String): Annotation = s.parseYaml.convertTo[Annotation]
```

Comment:
Actually, the toYaml method is already implemented, that's how its called on the class, so its better to leave this as is.